gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -90,7 +108,43 @@ def _merge_header_fields(self, first, second):
                            '{}, {} \n. Error: {}'.format(
                                first_value, second_value, str(e)))
 
-      first[second_key] = merged_value
+      first_headers[second_key] = merged_value
+
+  def _merge_conflicts(self, key, value, first_conflicts, second_conflicts):
+    # type: (str,
+    #        OrderedDict[str, Union[str, int]],
+    #        Dict[str, Set[str]],
+    #        Dict[str, Set[str]])-> None
+    """Updates ``first_conflicts``'s value of ``key`` by merging values from
+    ``first_conflicts`` and ``second_conflicts``.
+
+    If ``key`` does not already exist in ``first_conflicts``, then ``key`` is
+    not defined in the first headers.
+    If ``key`` does not already exist in ``second_conflicts``, it means that
+    ``value`` is the only definition for this key. Simply add ``value`` into the
+    ``second_conflicts``.
+    """
+    if not self._save_conflicts:
+      return
+    if key not in first_conflicts:
+      first_conflicts.update({key: set()})
+    if key not in second_conflicts:
+      second_conflicts.update({key: {self._format_conflict(value)}})
+    first_conflicts[key].update(second_conflicts[key])
+
+  def _format_conflict(self, field_definition):
```

Comment:
Is it for display purpose?  If so, maybe collect conflicts in a dict, and apply `format` on the whole dict at the very end when they are printed out.